release/src/main/scripts/mass_comment.py

```diff
@@ -27,42 +27,115 @@
+    "Run Dataflow Streaming ValidatesRunner",
+    "Run Dataflow ValidatesRunner Java 11",
+    "Run Dataflow ValidatesRunner Java 17",
     "Run Dataflow ValidatesRunner",
+    "Run Direct Runner Nexmark Tests",
+    "Run Direct ValidatesRunner Java 11",
+    "Run Direct ValidatesRunner Java 17",
+    "Run Direct ValidatesRunner in Java 11",
+    "Run Direct ValidatesRunner",
+    "Run Flink Runner Nexmark Tests",
+    "Run Flink ValidatesRunner Java 11",
     "Run Flink ValidatesRunner",
-    "Run Samza ValidatesRunner",
-    "Run Spark ValidatesRunner",
-    "Run Java Spark PortableValidatesRunner Batch",
-    "Run Python Spark ValidatesRunner",
-    "Run Python Dataflow ValidatesContainer",
-    "Run Python Dataflow ValidatesRunner",
-    "Run Python Flink ValidatesRunner",
-    "Run Twister2 ValidatesRunner",
-    "Run Python 3.6 PostCommit",
-    "Run Python 3.7 PostCommit",
-    "Run Python 3.8 PostCommit",
-    "Run SQL PostCommit",
+    "Run Go Flink ValidatesRunner",
+    "Run Go PostCommit",
     "Run Go PreCommit",
+    "Run Go Samza ValidatesRunner",
+    "Run Go Spark ValidatesRunner",
+    "Run GoPortable PreCommit",
+    "Run Java 11 Examples on Dataflow Runner V2",
+    "Run Java 17 Examples on Dataflow Runner V2",
+    "Run Java Dataflow V2 ValidatesRunner Streaming",
+    "Run Java Dataflow V2 ValidatesRunner",
+    "Run Java Examples on Dataflow Runner V2",
+    "Run Java Examples_Direct",
+    "Run Java Examples_Flink",
+    "Run Java Examples_Spark",
+    "Run Java Flink PortableValidatesRunner Streaming",
+    "Run Java Portability examples on Dataflow with Java 11",
+    "Run Java PostCommit",
     "Run Java PreCommit",
+    "Run Java Samza PortableValidatesRunner",
+    "Run Java Spark PortableValidatesRunner Batch",
+    "Run Java Spark v2 PortableValidatesRunner Streaming",
+    "Run Java Spark v3 PortableValidatesRunner Streaming",
+    "Run Java examples on Dataflow Java 11",
+    "Run Java examples on Dataflow Java 17",
+    "Run Java examples on Dataflow with Java 11",
     "Run Java_Examples_Dataflow PreCommit",
-    "Run JavaPortabilityApi PreCommit",
+    "Run Java_Examples_Dataflow_Java11 PreCommit",
+    "Run Java_Examples_Dataflow_Java17 PreCommit",
+    "Run Java_PVR_Flink_Batch PreCommit",
+    "Run Java_PVR_Flink_Docker PreCommit",
+    "Run Javadoc PostCommit",
+    "Run Jpms Dataflow Java 11 PostCommit",
+    "Run Jpms Dataflow Java 17 PostCommit",
+    "Run Jpms Direct Java 11 PostCommit",
+    "Run Jpms Direct Java 17 PostCommit",
+    "Run Jpms Flink Java 11 PostCommit",
+    "Run Jpms Spark Java 11 PostCommit",
+    "Run PortableJar_Flink PostCommit",
+    "Run PortableJar_Spark PostCommit",
     "Run Portable_Python PreCommit",
-    "Run PythonLint PreCommit",
+    "Run PostCommit_Java_Dataflow",
+    "Run PostCommit_Java_DataflowV2",
+    "Run PostCommit_Java_Hadoop_Versions",
+    "Run Python 3.5 PostCommit Pytest",
+    "Run Python 3.6 PostCommit Pytest",
```

Comment:
Whats the difference between this and the non-pytest postcommit

Reply:
Oh I meant to remove the pytest ones, thanks for noticing thhis.

I think these are old test jobs that should have been disabled. But they were still enabled when I ran my script, so they got pulled in.

Reply:
(I went ahead and disabled them earlier as well: https://ci-beam.apache.org/job/beam_PostCommit_Python35_Pytest_PR/)